code/jupyter_notebook/.ipynb_checkpoints/keys_and_addresses_python_notebook-checkpoint.ipynb

```diff
@@ -33,8 +33,8 @@
     "\n",
     "# Vitalik Buterin's Python Library for Bitcoin\n",
     "# No longer maintained!\n",
-    "# https://pypi.python.org/pypi/bitcoin/1.1.42\n",
-    "import bitcoin\n",
+    "# https://pypi.python.org/pypi/Bitcoin/1.1.42\n",
```

Comment:
small b

Reply:
A jupyter notebook is pure code, please be so careful with code changes, and please retest it with the proposed change.